app/validators/url_validator.rb

```diff
@@ -0,0 +1,24 @@
+class UrlValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    uri = URI.parse(value)
+    record.errors.add(attribute, 'is invalid') unless valid_url?(uri)
+  end
+
+  private
+
+  def valid_url?(uri)
+    uri && valid_host?(uri.host)
+  rescue URI::InvalidURIError
+    false
+  end
+
+  def valid_host? host
```

Comment:
I doubt that. Rubocop follows [bbatsov's styleguide](https://github.com/bbatsov/ruby-style-guide), which prefers parentheses all the time, and I don't remember seeing a warning about this even though I always use parens around single args.

Maybe you're thinking about the case where there are no args?